src/styles/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
```

Comment:
it s better to apply this property to all elements (use * selector)
when all elements have the same `box-sizing` value, their behavior becomes more predictable